packages/@cdktf/provider-generator/lib/get/generator/provider-generator.ts

```diff
@@ -168,20 +169,35 @@ export class TerraformProviderGenerator {
       }
       providerResource.providerVersion = providerVersion;
       files.push(this.emitResource(providerResource));
+      this.emitResourceReadme(providerResource);
     }
 
     this.emitIndexFile(name, files);
   }
 
+  private emitResourceReadme(resource: ResourceModel): void {
+    const filePath = `${resource.namespaceFolderPath}/README.md`;
```

Comment:
Should we use `path.join` here too?

Reply:
I briefly thought about it, but refrained from doing so, because the rest of this file is already only doing it manually 😄 And I didn't want to accidentally break things (by refactoring those bits as well then).

Reply:
But I can add this improvement if you want?